windows/rke2-quickstart.ps1

```diff
@@ -0,0 +1,105 @@
+<#
+.SYNOPSIS
+  A quickstart script to Setup and Install standalone RKE2 in Windows to be used as Worker Nodes.
+  This script enables features, sets up environment variables and adds default configuration that are needed to install rke2 in Windows and join a cluster
+.DESCRIPTION
+  Run the script to setup and install all RKE2 related needs and to join a cluster.
+.Parameter ServerIP
+    Server IP of Primary server where rke2 is already installed and the worker will join.`
+.Parameter Token
+    Token of Primary server.`
+.Parameter Version
+    Version of rke2 to download from github.`
+.Parameter Commit
+    Commit of RKE2 to download from temporary cloud storage.
+    If set, this forces Method=tar in install script.
+    * (for developer & QA use only)`
+.EXAMPLE
+  Usage:
+    Invoke-WebRequest ((New-Object System.Net.WebClient).DownloadString('https://github.com/rancher/rke2/blob/master/windows/rke2-quickstart.ps1'))
+    ./rke2-quickstart.ps1 $ServerIP <server-IP> $Token <server-token> $Version <rke2-version>
+#>
+
+[CmdletBinding()]
+param (
+    [Parameter()]
+    [String]
+    $ServerIP,
+    [Parameter()]
+    [String]
+    $Version,
+    [Parameter()]
+    [String]
+    $Token,
+    [Parameter()]
+    [String]
+    $Commit
+)
+
+function Write-InfoLog() {
+    Write-Output "[INFO] $($args -join " ")"
+}
+
+function Write-WarnLog() {
+    Write-Output "[WARN] $($args -join " ")"
+}
+
+function Write-DebugLog() {
+    Write-Output "[DEBUG] $($args -join " ")"
+}
+
+# Set the version or commit based of cli
+function Set-Version() {
+    if ($Commit) {
+        $Version = "commit $($Commit)"
+    }
+    elseif ($Version) {
+        $Version = $Version
+    }
+    return $Version
+}
+
+function Enable-Features() {
+    Enable-WindowsOptionalFeature -Online -FeatureName Containers -All
+    Restart-Computer -Wait -For PowerShell -Timeout 300 -Delay 2
```

Comment:
It looks like prompting to reboot is actually the default, so unconditionally rebooting probably isn't the right answer.

https://learn.microsoft.com/en-us/powershell/module/dism/enable-windowsoptionalfeature?view=windowsserver2022-ps

Reply:
So when rke2 joins successfully, the system reboots automatically.. removing this Restart command should be fine then?

Reply:
I have tested with both with and without this command (single node and HA) and was successfully able to join the windows node to the cluster

Reply:
Removed this Restart command. Will address any issues if observed on a separate PR